tools-local/procdump/Eula.txt

```diff
@@ -0,0 +1,75 @@
+﻿Sysinternals Software License Terms
+These license terms are an agreement between Sysinternals (a wholly owned subsidiary of Microsoft Corporation) and you. Please read them. They apply to the software you are downloading from technet.microsoft.com/sysinternals, which includes the media on which you received it, if any. The terms also apply to any Sysinternals
+* updates,
+* supplements,
+* Internet-based services,
+* and support services
```

Comment:
do you also need procdump64.exe for X64?

Reply:
Procdump.exe has the x64 libraries in it.